test/jellyfish/component/hls/ets_helper_test.exs

```diff
@@ -32,61 +32,64 @@ defmodule Jellyfish.Component.HLS.EtsHelperTest do
 
   test "rooms managment" do
     room_id = UUID.uuid4()
-    {:error, :room_not_found} = EtsHelper.get_partial(room_id, @partial_name, @offset)
+    assert {:error, :room_not_found} == EtsHelper.get_partial(room_id, @partial_name, @offset)
```

Comment:
[Assert works with pattern match](https://hexdocs.pm/ex_unit/1.12.3/ExUnit.Assertions.html#assert/1), so changing `=` to `==` wasn't necessary.